themes/default/data/registry/packages/artifactory.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
```

Comment:
I pushed a commit to update the metadata file based on the latest `resourcedocsgen` tool from the `docs` repo. That ordered the properties in the file as well as added the required `repo_url` and `schema_file_path` properties to it.

Reply:
<3 thanks!